crates/bevy_asset/src/server/mod.rs

```diff
@@ -218,9 +218,9 @@ impl AssetServer {
     }
 
     /// Retrieves the default [`AssetLoader`] for the given path, if one can be found.
-    pub async fn get_path_asset_loader<'a>(
+    pub async fn get_path_asset_loader<'a, 'b>(
         &self,
-        path: impl Into<AssetPath<'a>>,
+        path: impl Into<AssetPath<'b>>,
```

Comment:
I don't understand, doesn't this mean the same thing? Also `'a` is no longer used.